rl/api/Runner.py

```diff
@@ -108,10 +144,19 @@ def run(self) -> None:
             game_step += 1
 
             if len(actions) == 0 or game_step > self.max_game_len:
+                print(self.game_results)
+                if game_status == GameStates.ONGOING.__str__():

```

Comment:
może lepeij użyć .name zamiast .__str__() - trochę bardzie byłoby to przjeszyste

Reply:
Fixed